CONTRIBUTING.rst

```diff
@@ -48,7 +48,7 @@ Setting Up for Local Development
 ::
 
     # After activating your virtualenv
-    $ pip install -e '.[dev]'
+    $ pip install -e ".[dev]"
```

Comment:
It even works without any quotes on my machine (Linux). Would that work on Windows?

    pip install -e .[dev]

Reply:
Sure. I don't know why we put those quotes in the first place so I'd go for no quotes at all if it works everywhere.

Reply:
OK so that's why we put those quotes in the first place. Double quotes is fine, then.